DllImportGenerator/TestAssets/NativeExports/Arrays.cs

```diff
@@ -120,5 +121,18 @@ public static void Append(int** values, int numOriginalValues, int newValue)
             newArray[numOriginalValues] = newValue;
             *values = newArray;
         }
+
+        [UnmanagedCallersOnly(EntryPoint = "and_all_members")]
+        [DNNE.C99DeclCode("struct bool_struct;")]
+        public static byte AndAllMembers([DNNE.C99Type("struct bool_struct*")] BoolStructNative* pArray, int length)
```

Comment:
Absolutely thrilled with the fact we can test/demonstrate both sides of our custom marshalling in a single test!